scanpy/datasets/_datasets.py

```diff
@@ -377,6 +389,7 @@ def visium_sge(
         'Targeted_Visium_Human_ColorectalCancer_GeneSignature',
         'Parent_Visium_Human_ColorectalCancer',
     ] = 'V1_Breast_Cancer_Block_A_Section_1',
+    include_hires_tiff: bool = False,
```

Comment:
```suggestion
    ] = 'V1_Breast_Cancer_Block_A_Section_1',
    *,
    include_hires_tiff: bool = False,
```

I think it makes sense for this argument to be keyword only.